src/Search/Annotation/GearmanTaskDriver.php

```diff
@@ -58,16 +60,17 @@ public function registerWorkflow(Workflow $workflow)
         }
     }
 
-    public function addTasksToWorker(GearmanWorker $worker, LoggerInterface $logger)
+    public function addTasksToWorker(GearmanWorker $worker)
     {
         foreach ($this->tasks as $task) {
-            $this->addTaskToWorker($task, $worker, $logger);
+            $this->addTaskToWorker($task, $worker);
         }
     }
 
-    public function addTaskToWorker(GearmanTaskInstance $task, GearmanWorker $worker, LoggerInterface $logger)
+    public function addTaskToWorker(GearmanTaskInstance $task, GearmanWorker $worker)
     {
-        $worker->addFunction($task->name, Closure::bind(function (GearmanJob $job) use ($task, $logger) {
+        $worker->addFunction($task->name, Closure::bind(function (GearmanJob $job) use ($task) {
+            $this->logger->info('GearmanTaskDriver task started', ['task' => $task->name]);
```

Comment:
Also the first task for any importing is always validating the json so that we can be assured that it will continue through the tasks without failing at that step, narrowing down the potential causes.